helm/templates/backendconfig.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if eq .Values.global.cloudProvider "gcp" }}
```

Comment:
missing `.Values.ingress.deploy true`
both, gcp and ingress deploy checks are needed

Reply:
changed!